examples/AcidRain.hs

```diff
@@ -11,11 +11,11 @@ import Control.Monad.IO.Class (MonadIO(liftIO))
 import Control.Monad.State (MonadState, get, modify)
 import Data.Function ((&))
 import Streamly.Data.Stream (Stream)
-import Streamly.Data.Stream.Concurrent (MonadAsync)
+import Streamly.Data.Stream.Prelude (MonadAsync)
```

Comment:
Let's remove the import of "Streamly.Data.Stream" wherever we are importing "Streamly.Data.Stream.Prelude".